assets/airports/terrain/kpvd.geojson

```diff
@@ -0,0 +1,7 @@
02, 41.0363065 ], [ -71.5861602, 41.0323067 ], [ -71.5806602, 41.0263065 ], [ -71.5781587, 41.0213065 ], [ -71.5766602, 41.0156763 ], [ -71.5726604, 41.0223065 ], [ -71.5671602, 41.0273065 ], [ -71.5591602, 41.0308067 ], [ -71.5486602, 41.0313066 ], [ -71.5431602, 41.0298065 ], [ -71.5371602, 41.0263077 ], [ -71.5301595, 41.0183065 ], [ -71.5271602, 41.0110793 ], [ -71.5231606, 41.0178065 ], [ -71.5176602, 41.0228067 ], [ -71.5101602, 41.0263065 ], [ -71.5021602, 41.0273065 ], [ -71.4931602, 41.0253068 ], [ -71.4871602, 41.0218078 ], [ -71.4821595, 41.0168065 ], [ -71.4776602, 41.0080516 ], [ -71.4741606, 41.0143065 ], [ -71.4671602, 41.0208065 ], [ -71.4601602, 41.0238071 ], [ -71.4511602, 41.0248065 ], [ -71.4441602, 41.0233066 ], [ -71.4356602, 41.0183065 ], [ -71.4306578, 41.0123065 ], [ -71.4281602, 41.0070309 ], [ -71.4251609, 41.0128065 ], [ -71.4191602, 41.019307 ], [ -71.4131602, 41.0228067 ], [ -71.4046602, 41.0248066 ], [ -71.3956602, 41.0238066 ], [ -71.3881602, 41.0203065 ], [ -71.3831599, 41.0158065 ], [ -71.3781602, 41.0083241 ], [ -71.3746611, 41.0158065 ], [ -71.3696602, 41.0213068 ], [ -71.3616602, 41.0258067 ], [ -71.3536602, 41.0273065 ], [ -71.3461602, 41.0263068 ], [ -71.3391602, 41.0233065 ], [ -71.3321598, 41.0168065 ], [ -71.3286602, 41.0109324 ], [ -71.325661, 41.0188065 ], [ -71.3196602, 41.0258069 ], [ -71.3151602, 41.0288065 ], [ -71.3076602, 41.0313065 ], [ -71.2991602, 41.0313065 ], [ -71.2906602, 41.0283066 ], [ -71.28266, 41.0213065 ], [ -71.2796602, 41.016008 ], [ -71.2756635, 41.0258065 ], [ -71.2711602, 41.0313066 ], [ -71.2641602, 41.0358065 ], [ -71.2561602, 41.0378066 ], [ -71.2466602, 41.0368066 ], [ -71.2386602, 41.0328065 ], [ -71.2336598, 41.0278065 ], [ -71.2291601, 41.0203065 ], [ -71.2266595, 41.0138065 ], [ -71.2256602, 41.0083177 ], [ -71.2231602, 41.0163065 ], [ -71.2201608, 41.0208065 ], [ -71.2161602, 41.0248066 ], [ -71.2101602, 41.0283069 ], [ -71.2026602, 41.0303065 ], [ -71.1946602, 41.0298066 ], [ -71.1861602, 41.0263067 ], [ -71.1796587, 41.0203065 ], [ -71.1766602, 41.0153579 ], [ -71.1746607, 41.0243065 ], [ -71.1701602, 41.0318065 ], [ -71.1641602, 41.0368068 ], [ -71.1546602, 41.0403066 ], [ -71.1466602, 41.0403065 ], [ -71.1426602, 41.0393065 ], [ -71.1351602, 41.0353065 ], [ -71.13066, 41.0308065 ], [ -71.1246561, 41.0218065 ], [ -71.1211602, 41.0123083 ], [ -71.1181602, 41.0233065 ], [ -71.1156602, 41.0273065 ], [ -71.1091602, 41.0333067 ], [ -71.1041602, 41.035807 ], [ -71.0986602, 41.0373065 ], [ -71.0916602, 41.0373066 ], [ -71.0856602, 41.0358065 ], [ -71.0781602, 41.0313066 ], [ -71.0736601, 41.0263065 ], [ -71.0676568, 41.0178065 ], [ -71.0641602, 41.0088067 ], [ -71.0636603, 41.0158065 ], [ -71.0621602, 41.0208065 ], [ -71.0586603, 41.0268065 ], [ -71.0541602, 41.0313067 ], [ -71.0481602, 41.0348069 ], [ -71.0406602, 41.0368065 ], [ -71.0351602, 41.0368065 ], [ -71.0295706, 41.0353098 ], [ -71.0294444, 40.9636111 ], [ -71.8525, 40.7647222 ], [ -72.0326161, 40.9328514 ] ] ] } }
```

Comment:
![image](https://cloud.githubusercontent.com/assets/5103735/25863696/c98952be-34ba-11e7-89ab-3f93984b861a.png)

Something's up with the terrain... :confused: